builtin/credential/radius/path_config.go

```diff
@@ -0,0 +1,214 @@
+package radius
+
+import (
+	"strings"
+
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `config`,
```

Comment:
Can we use a single line string bounds here instead of backtick? Nitpick, sorry.